### DISCUSSION TITLE
What is Permissions tab in Data Importer plugin

### DISCUSSION THREAD

**Discussion by anshu1thakur** [pimcore/pimcore#16583] [General]
Hi , There is a permision tab in Data importer plugin where we can add users and roles permission but the use of it because using it is not making any change for any user and on the other hand other users can also make changes into this permission tab

**Comment by kingjia90** [comment]
Need more details on how to reproduce it (maybe in a ticket within the affected repository), but by quickly looking at it, it seems that is if you set `plugin_datahub_admin`
https://github.com/pimcore/data-hub/blob/5456da0e63bff8b3da18c0592a0b658f8132119b/src/Configuration.php#L448

you are always allowed to do anything in every data-hub related bundle including data importer
https://github.com/pimcore/data-importer/blob/fa5849b38462989a64dfa42841c3cee1d0e00e1b/src/Controller/ConfigDataObjectController.php#L89

Could you please check if that is flagged?
